### ISSUE TITLE
Search input boxes lack labels

### ISSUE THREAD

**Issue by jonels-msft** [readthedocs/sphinx_rtd_theme#970]
### Problem

For accessibility, form inputs should have corresponding labels. The search box above the table of contents, and the search box inside the version popup menu aren't associated with labels.

### Reproducible Project

https://pg-auto-failover.readthedocs.io

#### Error Logs/Results

![label-1](https://user-images.githubusercontent.com/48225118/86639358-5fba3c00-bf9e-11ea-987a-26f629f2f3c2.JPG)

![label-2](https://user-images.githubusercontent.com/48225118/86639368-634dc300-bf9e-11ea-8e09-bebbeaca6a6f.JPG)

#### Expected Results

Fix any of the following:
* aria-label attribute does not exist or is empty
* aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
* Form element does not have an implicit (wrapped) `<label>`
* Form element does not have an explicit `<label>`
* Element has no title attribute or the title attribute is empty

See [WCAG 3.3.2](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)

### Environment Info

- Python Version: 3.7.3
- Sphinx Version: 1.8.5
- RTD Theme Version: 0.4.3
